packages/core/upload/admin/src/components/AssetDialog/BrowseStep/index.js

```diff
@@ -66,6 +68,16 @@ export const BrowseStep = ({
   selectedAssets,
 }) => {
   const { formatMessage } = useIntl();
+
+  const { data: currentFolder, isLoading: isCurrentFolderLoading } = useFolder(
```

Comment:
Do you think you could at least add one test to verify that `canRead` is properly taken into account here and that `useFolder` is not called? If you want you could just copy the one from https://github.com/strapi/strapi/pull/13902 and adjust it to your needs.

Reply:
nice catch, done!
I didn't add any test related to Breadcrumbs since I added tests for the component itself, does that sound good to you?